hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -32,8 +32,8 @@ sudo rpi-eeprom-update -d -f ./pieeprom-new.bin
 sudo reboot
 ```
 
-## Configuration Items
-This section describes all the configuration items available in the bootloader.
+## Configuration Properties
+This section describes all the configuration items available in the bootloader. The syntax is the same as [config.txt](../../configuration/config-txt/) but the properties are specific to the bootloader. [Conditional filters](../../configuration/config-txt/conditional.md) are also supported except for EDID.
```

Comment:
I guess the `[pi2]` filter isn't of much use though :wink:   Are the `gpio` filters available, allowing you to dynamically select which parts of the EEPROM config get used? That's pretty funky if so.